client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -242,6 +242,103 @@ body {
   float: right;
 }
 
+.visual-card {
+  background: #FFFFFF;
+  border: 1px solid fade(@redash-gray, 15%);
+  border-radius: 3px;
+  margin: 5px;
+  width: 212px;
+  padding: 15px 5px;
+  cursor: pointer;
+  box-shadow: none;
+  transition: transform 0.12s ease-out;
+  transition-duration: 0.3s;
+  transition-property: box-shadow;
+
+  display: flex;
+  align-items: center;
+
+  &:hover {
+    box-shadow: rgba(102, 136, 153, 0.15) 0px 4px 9px -3px;
+  }
+
+  img {
+    width: 64px !important;
+    height: 64px !important;
+    margin-right: 5px;
+  }
+
+  h3 {
+    font-size: 13px;
+    color: #323232;
+    margin: 0 !important;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+}
+
+.visual-card--selected {
+  background: fade(@redash-gray, 3%);
+  border: 1px solid fade(@redash-gray, 15%);
+  border-radius: 3px;
+  padding: 0 15px;
+  box-shadow: none;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  justify-content: space-around;
+  margin-bottom: 15px;
+  width: 100%;
+
+  img {
+    width: 64px;
+    height: 64px;
+  }
+
+  a {
+    cursor: pointer;
+  }
+}
+
+@media (max-width: 1200px) {
+  .visual-card {
+    width: 217px;
+  }
+}
+
+@media (max-width: 755px) {
```

Comment:
This is how it looks for me at 755px.

![Screen Shot 2019-04-07 at 9 51 38](https://user-images.githubusercontent.com/486954/55679827-c889b300-591a-11e9-9c05-95cac5d19433.png)

Reply:
I think you can just keep all at 217px without any adjustments to smaller viewports.

Reply:
I guess this one was due to the unnecessary change in DOM with both `<a>` and `<div>`. My bad on missing it :sweat_smile:. Now:

![data-source-list](https://user-images.githubusercontent.com/3356951/55690360-fbf22f00-5966-11e9-9ab6-d23b92181303.gif)